qemu/tests/cfg/hotplug_mem_migration.cfg

```diff
@@ -0,0 +1,92 @@
+- hotplug_mem_migration:
+    type = hotplug_mem_migration
+    mem_fixed = 4096
+    smp = 4
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
+    login_timeout = 600
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
+    no Host_RHEL.6 Host_RHEL.7 Host_RHEL.8
+    no RHEL.6 RHEL.7 RHEL.8
+    no Windows
+    threshold = 0.11
+    # Notes:
+    #    The threshold on arm and ppc was confirmed with developer
+    #    since crashkernel size is much bigger than x86
+    aarch64,ppc64,ppc64le:
+        threshold = 0.15
+    sub_type = migration
+    max_vms = 2
+    migration_test_command = help
+    migration_bg_command = "cd /tmp; nohup tcpdump -q -i any -t ip host localhost"
+    migration_bg_check_command = pgrep tcpdump
+    migration_bg_kill_command = pkill -9 tcpdump
+    kill_vm_on_error = yes
+    iterations = 2
+    mig_timeout = 1200
+    ping_pong = 1
+    migration_protocol = "tcp"
+    target_mems = "plug1 plug2"
+    cmd_check_online_mem = lsmem
+    cmd_new_folder = ' rm -rf /tmp/numa_test/ && mkdir /tmp/numa_test'
+    numa_test = 'numactl -m %s dd if=/dev/urandom of=/tmp/numa_test/test '
+    numa_test += 'bs=1k count=2621440 && rm -rf /tmp/numa_test/'
+    stress_args = '--cpu 4 --io 4 --vm 2 --vm-bytes 2048M'
+    variants numa_nodes:
+        - one:
+            guest_numa_nodes = "node0"
+            mem_devs = "memN0"
+            use_mem_memN0 = "no"
+            size_mem_memN0 = 4096M
+            backend_mem_memN0 = memory-backend-ram
+            numa_memdev_node0 = mem-memN0
+            del numa_mem
+            del numa_cpus
+            del numa_nodeid
+        - two:
+            guest_numa_nodes = "node0 node1"
+            del numa_mem
+            del numa_cpus
+            numa_nodeid = 0
+            mem_devs = "memN0 memN1"
+            numa_memdev_node0 = mem-memN0
+            numa_memdev_node1 = mem-memN1
+            use_mem_memN0 = "no"
+            use_mem_memN1 = "no"
+            size_mem_memN0 = 2048M
+            size_mem_memN1 = 2048M
```

Comment:
```
x86_64:
    size_mem_memN0 = 4096M
    size_mem_memN1 = 4096M
```
You can include it for x86_64

Reply:
Thank a lot for your review.
Considering that the kernel will take up more and more memory in the future, 
I increased the mem size of all platforms and simultaneously increased the pressure value.